### DISCUSSION TITLE
Firefly Apple IOS App

### DISCUSSION THREAD

**Discussion by schmaecker** [iotaledger/firefly#7685] [Help & Support]
Hi, i made a mistake and send all IOTAs to the old version of Firefly Wallet App on Apple Iphone. Now i cant see my Funds but the tansaction was correct. Transaction ID: 

0x387b5f4f069e4f0759014fee77eae4b91915b80bec589f80f30df59b32944317

 Is there any possibillity to get the IOTAs on Desktop (same Acoount).

With the Update, i have a new Adress to receive IOTAs. 

Tryed everything already. Nodes are not accepted on the desktop...

Let me know if you need some more informations. Thanks a lot!

**Comment by begonaalvarezd** [comment]
Just use the balance finder so that your firefly desktop version finds that address you used, go to settings > advanced > balance finder. Keep searching until all your funds are found

**Reply by schmaecker** [reply]
Sometimes its too easy. Took 5h! Thanks a lot!!

**Reply by begonaalvarezd** [reply]
@schmaecker Did you find your funds 🤞🏼 ? If so, I will proceed and close the discussion, thanks
